packages/flutter_tools/lib/src/ios/mac.dart

```diff
@@ -167,6 +168,7 @@ Future<XcodeBuildResult> buildXcodeProject({
     RemoveBitcodeMigration(app.project, globals.logger),
     XcodeThinBinaryBuildPhaseInputPathsMigration(app.project, globals.logger),
     UIApplicationMainDeprecationMigration(app.project, globals.logger),
+    MetalAPIValidationMigrator(app.project, globals.logger),
```

Comment:
This only migrates iOS, you'll also want to add it here for macOS:
https://github.com/flutter/flutter/blob/364e4be81638a3f109820135c32ffdd9d8ff87c9/packages/flutter_tools/lib/src/macos/build_macos.dart#L105

Reply:
Done